libpod/container_internal.go

```diff
@@ -1308,8 +1308,8 @@ func (c *Container) mountStorage() (_ string, deferredErr error) {
 		}
 		defer func() {
 			if deferredErr != nil {
-				if err := c.unmountSHM(c.config.ShmDir); err != nil {
-					logrus.Errorf("Error unmounting SHM for container %s after mount error: %v", c.ID(), err)
+				if err := c.unmountExtra(c.config.ShmDir); err != nil {
+					logrus.Errorf(err.Error())
```

Comment:
It think it's worth preserving the wrapping, especially as we can track it do this location in the code.

Reply:
The wrapping is happening at a lower level.  Basicaly umountExtra will umount the shm as well as any overlay mount, if these fail, then they will be wrapped.